src/components/molecules/NavBar/playa.scss

```diff
@@ -16,9 +16,10 @@ $page-max-width: 1240px;
 
 $sand: #937c63;
 
+// @debt Refactor the relevant parts of this file into ./NavBar.scss
```

Comment:
[Future Cleanup] `@debt` Refactor the relevant parts of this file into ./NavBar.scss